docs/mobile/repo-structure.md

```diff
@@ -0,0 +1,162 @@
+interact with.
+
+```text
+|- assets/
+|  |- images/
+|  |- datasets
+|- chapter1/
+|  |- lesson1.yml
+|  |- lesson2.yml
+|  |- practice.yml
+|- chapter2/
+|  |- lesson1.yml
+|  |- lesson2.yml
+|  |- practice.yml
+|- chapter3/
+|  |- lesson1.yml
+|  |- lesson2.yml
+|  |- practice.yml
+|- chapter4/
+|  |- lesson1.yml
+|  |- lesson2.yml
+|  |- practice.yml
+|- .gitignore
+|- .circleci
+|  |- config.yml
+|- README.md
+|- manifest.yml
+```
+
+An overview of what each of this does is:
+
+| File/Folder                   | Description                                                                                                       |
+| :--------------------         | :-------------------------------------------------------------------------                                        |
+| `assets*/`                    | Folder containing all datasets or images you may want to use in your lessons (eg. csv / python/ R / image files). |
+| `chapter*/`                   | Contains all the lessons for a particular chapter. This is a convention, not a strict requirement.                |
+| `.gitignore`                  | Files and folders which should be ignored by Git                                                                  |
+| `README.md`                   | A readme file with a list of resources more explanation to get started.                                           |
+| [`manifest.yml`](manifest.md) | A manifest file specifying metadata about the course, chapters, lessons, and assets.                              |
+
+## Assets
+
+This directory should contain all datasets or images you may want to use in your
+lessons (eg. csv / python/ R / image files). You can organize it any way you
+like, but sub-directories for images, datasets, etc. is one logical strategy.
+
+## Chapters
+
+Each chapter should belong to its own dedicated subdirectory. You can name
+lessons according to their subject matter or ordinality at your own discretion.
+
+## README
+
+The course README should explain the objective of the course and outline the
+curriculum at a high level.
+
+## .gitignore
+
+[Typical `.gitignore` file](https://www.atlassian.com/git/tutorials/gitignore).
+
+## .circleci/config
+
+The `.circleci/config.yml` file is the configuration for DataCamp's course build
+process. It triggers the building and deployment of your course when you perform
+certain actions, like merging into `development` / `master` branches, or tagging
+a release. See [the development process documentation](development.md) for more
+details.
+
+## Manifest
+
+The manifest file `manifest.yml` is what ties everything together. It lays out
+the locations of your chapters, lessons, and assets. For example:
+
+```yaml
+authoringVersion: 1
+title: Introduction to SQL
+description: "Master the basics of querying databases with SQL, the world's most popular databasing language."
+key: e738342c-15ca-4866-8786-500214d9a363
```

Comment:
Are the keys auto-generated? Might be useful to specify that the instructor does not have to type random strings of letters and numbers.

Reply:
Would that it were so simple. They are not auto generatred because we have no authoring front end :(